### PR TITLE
Cleanup & Update

### PR DESCRIPTION
**Pull request by GoliathLabs**

### CODE REVIEW

**Comment by GoliathLabs** [comment]
Removed all unused packages! 😄

**Comment by GoliathLabs** [comment]
You also might want to change the repository description to match the README.md: "A repository for our Freifunk Munich specific packages of the Gluon firmware we use in our images."

File: ffmuc-autoupgrade-experimental2testing/files/lib/gluon/upgrade/505-move-experimental2testing [hunk 1]
```diff
@@ -1,11 +0,0 @@
-#!/usr/bin/lua
-
-local uci = require("simple-uci").cursor()
-
-local branch = uci:get('autoupdater', 'settings', 'branch')
-
-if "experimental" == branch then
-    uci:set('autoupdater', 'settings', 'branch', 'testing')
```
Comment (krombel) [inline ffmuc-autoupgrade-experimental2testing/files/lib/gluon/upgrade/505-move-experimental2testing]: I would like to "keep" this package for future reference.
Context: We have the stale wireguard branch which I would like to migrate to experimental via this approach (and there might be next we might want to use this code with in future as well)

Comment (GoliathLabs) [reply ffmuc-autoupgrade-experimental2testing/files/lib/gluon/upgrade/505-move-experimental2testing]: Resolved

**Pull request closed without merging**